sdk/voiceservices/ci.yml

```diff
@@ -0,0 +1,32 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
```

Comment:
refer https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/edgeorder/ci.mgmt.yml to fix this file for ci pipeline, and rename this file to `ci.mgmt.yml`

Reply:
Ok, fixed.

For reference, I was following the docs here: https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=sdk-generation-from-scratch, which have:

> Create a ci.yml file under the root folder of this SDK which will be used to create the CI pipeline. Here the folder is root\sdk\servicelinker, and the content in this file is:

(and then an example file). Is it worth someone correcting those docs?